code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -72,7 +72,7 @@
 
 			if (src.client)
 				if (client.prefs.muted & MUTE_IC)
-					src << "\red You cannot send IC messages (muted)."
+					src << "<span class='warning'>You cannot send IC messages (muted).</span>"

```

Comment:
I think all muted messages should perhaps be `danger`? The one above this one was.